data/meta/tokens/1/0xfeBb0bbf162E64fb9D0dfe186E517d84C395f016.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Balancer Aave v3 Boosted StablePool",
+  "symbol": "bb-a-USD",
+  "description": "This token represents a Balancer Boosted liquidity pool which is composed of 3 other nested balancer pools. Holders earn fees from users trading in the pools, and can also deposit the LP to Balancer's gauges to earn BAL emissions. This pool contains DAI & aDAI, USDC & aUSDC, USDT & aUSDT, the pool uses Aave for yield a decentralized money market on Ethereum.",
```

Comment:
IMO the ending of the last sentence in the description could have its organization improved. See idea below.

> the pool uses Aave for yield a decentralized money market on Ethereum.

could maybe be changed to 

> the pool uses Aave, a decentralized money market on Ethereum, to generate yield.